app/models/common_page.rb

```diff
@@ -0,0 +1,44 @@
+# Page model for pages with content shared by all modules
+# whilst still able to render training_module/module_item specific variables
+#
+# Pagination is disabled for these pages
+#
+class CommonPage < ContentPage
+  # @return [String]
+  def heading
+    I18n.t('heading', scope: type)
+  end
+
+  # @return [String]
+  def body
+    I18n.t('body', scope: type, **locals)
+  end
+
+  # @return [false]
+  def page_numbers?
+    false
+  end
+
+private
+
+  # @return [Hash]
+  def locals
```

Comment:
should this be in the model?

Reply:
Is there alternative? We are using it to collate values defined at the module and item level.